src/js/media/html5.js

```diff
@@ -408,6 +415,8 @@ vjs.Html5.prototype.addRemoteTextTrack = function(options) {
 
   this.el().appendChild(track);
 
+  this.addCrossorigin();
```

Comment:
Yep, if you add a remote text track after a video has loaded, we'd want to add the `crossorigin` attribute to video element before adding in the track as we don't have the `loadstart` opportunity for it.